gtsfm/configs/deep_front_end.yaml

```diff
@@ -37,9 +37,9 @@ SceneOptimizer:
     bundle_adjust_2view_maxiters: 100
 
     verifier:
-      _target_: gtsfm.frontend.verifier.ransac.Ransac
+      _target_: gtsfm.frontend.verifier.pytheia_verifier.PyTheiaVerifier
       use_intrinsics_in_verification: True
-      estimation_threshold_px: 4 # for H/E/F estimators
+      estimation_threshold_px: 6.0 # for H/E/F estimators
```

Comment:
did you try keeping this threshold the same? I suspect the "num_image_pairs_all_gt_inliers" regression (and potentially downstream regressions) might be because of this change. its allowing some bad correspondences?